src/main/java/fr/openent/competences/constants/DateFormat.java

```diff
@@ -0,0 +1,9 @@
+package fr.openent.competences.constants;
+
+public class DateFormat {
+    public static final String SQL_FORMAT = "yyyy-MM-dd'T'HH:mm:ss";
+    public static final String MONGO_FORMAT = "yyyy-MM-dd HH:mm:ss";
+    private DateFormat() {
+        throw new IllegalStateException("Utility class");
```

Comment:
not necessary

Reply:
yes it is => https://rules.sonarsource.com/java/RSPEC-1118/